taccsite_cms/templates/guides/getting_started.tam.html

```diff
@@ -0,0 +1,29 @@
+{% extends "guide.html" %}
+{% load cms_tags staticfiles tacc_uri_shortcuts %}
+
+{% block guide %}
+  {% site_uri as site_uri %}
+  <style>
+  .nav-link-ref {
+    white-space: nowrap;
+  }
+  .nav-link-ref .icon {
+    vertical-align: text-bottom;
+  }
+  </style>
+  <div class="row">
+    <div class="col">
+      <h1>Getting Started</h1>
+
+      <h2>Account Holders</h2>
+
+      <p>If you already have an account, you may log in by clicking the <q class="nav-link-ref" href="/login"><i class="icon icon-user"></i> Log in</q> link at the top right of the page.</p>
+
+      <h2 id="create-account">Creating an Account</h2>
+
+      <p>Before you can access the Portal, you must first have an active account with TACC.</p>
+
+      <p>If you are a new TACC user, you may <a href="{{ site_uri.absolute_uri }}tam/register" target="_blank">sign up for a new TACC account</a>.</p>
```

Comment:
[Redirect created on APCD CMS (prod and staging).](https://github.com/TACC/Core-CMS/pull/549#issuecomment-1230805609)